<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddConfiguration</class>
 <widget class="QDialog" name="AddConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_ram">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>RAM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_gpu">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>GPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_cpu">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>CPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_storage">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Storage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_blockPower">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>BP</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox_ram">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_gpu"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_cpu"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_storage"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_blockPower"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_computerName">
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Компьютер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_computerName"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="button_deleteConfiguration">
       <property name="text">
        <string>Удалить конфигурацию</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_configure">
     <property name="text">
      <string>Сконфигурировать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
